### ISSUE TITLE
Compiles commented code into result CSS on a large project

### ISSUE THREAD

**Issue by plagueTech** [scout-app/scout-app#322]
**OS:** Debian 8
**Scout-App Version:** 2.12.12
**Information from Dev Tools:** No warnings or error messages.

When working on a Grav theme, I noticed my overrides were not being honored. So I commented out the parts of the base .scss files I didn't want. Those rules were still compiled into the final css file by Scout-App.

**Comment by TheJaredWilcurt** [comment]
@plagueTech 

Can I get a screenshot of your folder/file structure. Or if you can zip up your sass folder and upload it, I can attempt to reproduce the issue.

**Comment by plagueTech** [comment]
[scoutPossibleBug.tar.gz](https://github.com/scout-app/scout-app/files/795167/scoutPossibleBug.tar.gz)

Here's an archive of the entire scss directory from the theme in question. This is the largest set of stylesheets I've put Scout up against.

**Comment by TheJaredWilcurt** [comment]
Tested on Debian 8 and Windows 7. Unable to reproduce.

Here's a video of me trying to reproduce the issue:

* https://youtu.be/MvfegVVrAwk

**Comment by plagueTech** [comment]
Apologies for my vagueness in my initial comment! I had the issue in Crunchbang 64-bit so your debian 8 test environment is spot on. 

Specifically, it was the file /template/_header.scss.

In the archive I sent, I believe the styles in the #header selector at the top of the file are still commented out. Those styles are what I was trying to override in my custom.scss but after compile the original rules were being honored. Even after dumping my browser cache and recompiling several times and dumping the cache in the Grav CMS as well.

After watching your video, you did a lot to try and make it happen and it didn't. That makes me think it's an issue with my system instead of Scout itself. The way I got around it was to eliminate the _header.scss from the equation altogether and just add the styles I wanted from it to my custom stylesheet. And then after compiling everything once, I reduced the scope of Scout to just my custom stylesheet. After I did that I didn't have a problem getting my custom styles to take priority.

Thanks for putting attention towards the issue.
